### PR TITLE
Fix reading of LSB multiplexor

### PR DESCRIPTION
**Pull request by stephane**

### CODE REVIEW

**Comment by FHenon** [comment]
Est-ce que cela peut résoudre le problème de mauvaise lecture de certaines trames des bus12m ?

**Comment by stephane** [comment]
Yes indeed!

File: caneton/decode.py [hunk 1]
```diff
@@ -155,13 +154,14 @@ def message_decode(message_id, message_length, message_data, dbc_json):
         message_binary_length)
 
     if message_info.get('has_multiplexor', False):
-        multiplexing_mode = message_get_current_multiplexing_mode(
+        multiplexor = message_get_multiplexor(
```
Comment (rsebille) [inline caneton/decode.py:157]: Could the function be replaced by  `multiplexor = int(message_data[:8])`?
From what I see, only signals have an _endianess_ so the messages must have the multiplexor always in the same bits.

Comment (stephane) [reply caneton/decode.py:157]: multiplexor have _endianess_ too (there are just specialized signals). The only information we use in caneton is the value but as we export the function in our API I think it make sense to provide more information and a consistent API with the signal information.

Comment (rsebille) [reply caneton/decode.py:157]: Indeed, forget I says something :).

**Comment by FHenon** [comment]
Merge ?

**Pull request merged**